src/ciphers/transposition.rs

```diff
@@ -82,11 +79,6 @@ pub fn transposition(key: &str, input: &str) -> String {
 mod test {
     use super::*;
 
-    #[test]
-    fn test_word() {
-        assert_eq!("OMLERX", transposition("key", "lorem"));
-    }
```

Comment:
Since this tests a different word, let's keep it. Otherwise, let's add it to the example as well.